tx/textual/internal/utils/protoreflect_list.go

```diff
@@ -1,4 +1,4 @@
-package valuerenderer_test
+package utils
```

Comment:
How about:

```suggestion
package listpb
```

to be consistent with protobuf's anypb, timestamppb, durationpb...